.dev/sass/mixins/_global.scss

```diff
@@ -296,7 +296,7 @@ meta.foundation-data-attribute-namespace {
 }
 
 html,
-body { font-size: $base-font-size; height: 100%; }
+body { font-size: $base-font-size; }
```

Comment:
@EvanHerman Have you seen checked if this affected any part of the themes in any way. I wonder why height was there in the first place.